File: app/src/main/java/com/appspot/pistatium/ssmemo/adapters/MemoCellInterface.java
package com.appspot.pistatium.ssmemo.adapters;

import com.appspot.pistatium.ssmemo.models.Memo;

/**
 * Created by kimihiro on 15/11/17.
 */
public interface MemoCellInterface {
    void onClickFav(Memo memo);
    void onClickDelete(Memo memo);
}

File: app/src/main/kotlin/com/appspot/pistatium/ssmemo/interfaces/MemoCellInterface.kt
package com.appspot.pistatium.ssmemo.interfaces

import com.appspot.pistatium.ssmemo.models.Memo

/**
 * Created by kimihiro on 15/11/17.
 */
interface MemoCellInterface {
    fun onClickFav(memo: Memo)
    fun onClickDelete(memo: Memo)
}
